cellmlmanip/model.py

```diff
@@ -77,10 +77,13 @@ class Model(object):
     (e.g. units) about symbols used in those equations
 
     :param name: the name of the model e.g. from <model name="">
+    :param cmeta_id: the cmeta_id of the model e.g. from <model cmeta_id="">
     """
-    def __init__(self, name):
+    def __init__(self, name, cmeta_id):
 
         self.name = name
+        self.cmeta_id = cmeta_id
+        self.rdf_identity = rdflib.URIRef('#' + cmeta_id) if cmeta_id is not None else None
```

Comment:
I'd just say `if cmeta_id else` for simplicity! Also, you could use your feature in `create_rdf_node`:
```python
   self.rdf_identity = create_rdf_node('#' + cmeta_id) if cmeta_id else None
```